openedx/features/partners/models.py

```diff
@@ -48,13 +48,13 @@ class Partner(TimeStampedModel):
             _('Accepted extensions: .jpg, .png (maximum {mb} MB)'), mb=bytes_to_mb(PARTNER_IMAGE_MAX_SIZE)
         ), upload_to=UploadToPathAndRename(path='images', name_prefix='main_bg_image', add_path_prefix=True)
     )
-    heading1 = models.CharField(blank=True, max_length=100, help_text="Heading 1")
-    heading1_description = models.TextField(blank=True)
-    heading2 = models.CharField(blank=True, max_length=100, help_text="Heading 2")
-    heading2_description = models.TextField(blank=True)
+    heading1 = models.CharField(blank=True, max_length=100)
+    heading1_description = models.TextField(blank=True, max_length=500, help_text=_("max 500 characters"))
+    heading2 = models.CharField(blank=True, max_length=100)
+    heading2_description = models.TextField(blank=True, max_length=500, help_text=_("max 500 characters"))
```

Comment:
just a suggestion can we name them properly like `primary_heading` or `secondry_heading`.